src/Exceptionless.Web/Exceptionless.Web.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="Exceptionless.AspNetCore.Signed" Version="4.3.2004" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.4.0" />
     <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.2" />
+    <PackageReference Include="Joonasw.AspNetCore.SecurityHeaders" Version="2.7.0" /> 
```

Comment:
Seems to already exists below... Possible occurred through a merge-conflict or alike

Reply:
You are right. Thanks.